roles/internal/webserver-app/tasks/drupal.yml

```diff
@@ -62,3 +62,26 @@
   args:
     chdir: "{{ drupal_core_path }}"
   when: drupal_member_block.changed is defined and drupal_member_block.changed
+
+- name: Set default solr server to point to CLAW core
+  command: drush -y config-set search_api.server.default_solr_server backend_config.connector_config.core CLAW
+  args:
+    chdir: "{{ drupal_core_path }}"
+  register: set_search_api_config
+  changed_when: "'Do you want to update' in set_search_api_config.stdout"
+
+- name: Get solr config files to copy
+  command: "find /var/www/html/drupal/web/modules/contrib/search_api_solr/solr-conf/6.x -type f"
+  register: files_to_copy
+  changed_when: false
+
+- name: Copy solr config files
+  copy:
+    src: "{{ item }}"
+    dest: "/var/solr/data/CLAW/conf"
```

Comment:
Can you make this path also a configurable variable. Not sure if it is used anywhere else, but good to make it easier for people to put it where they want it.